app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="linear_padding">15dp</dimen>
+    <dimen name="text_size">20sp</dimen>
+    <dimen name="button_margin">40dp</dimen>
```

Comment:
Rename `linear_padding -> login_layout_padding`
Rename `button_margin  -> login_button_margin`

Reply:
Okay

Reply:
Please make this change in the same branch:
Follow these steps:
- git checkout pr/arynshaikh/TuT-103-create-login-screen
- git pull
- Make changes in Android Studio
- git diff
- git add .
- git commit
- git push origin HEAD